docs/blog/2018-10-09-vscode-gatsby-development/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Using VS Code for Supercharged Gatsby.js Development"
+author: Michael Holtzman
+date: 2018-10-09
```

Comment:
We'll need to update this for the date it's published. Maybe we shoot for Thursday or Friday @lindaleebumblebee?